README.md

```diff
@@ -1,89 +1,88 @@
 # hoodie-task-server
 
-> CouchDB-based REST & front-end API for asynchronous background tasks
+> Task API
 
-[![Build Status](https://travis-ci.org/hoodiehq/hoodie-task-server.svg?branch=master)](https://travis-ci.org/hoodiehq/hoodie-task-server)
-[![Coverage Status](https://coveralls.io/repos/hoodiehq/hoodie-task-server/badge.svg?branch=master)](https://coveralls.io/r/hoodiehq/hoodie-task-server?branch=master)
-[![Dependency Status](https://david-dm.org/hoodiehq/hoodie-task-server.svg)](https://david-dm.org/hoodiehq/hoodie-task-server)
-[![devDependency Status](https://david-dm.org/hoodiehq/hoodie-task-server/dev-status.svg)](https://david-dm.org/hoodiehq/hoodie-task-server#info=devDependencies)
+`hoodie-task-server` is a [Hapi](http://hapijs.com/) plugin that exposes an API for managing tasks.
```

Comment:
it also exposes routes. Maybe that’s what you mean with `API`, but maybe we can clarify this? To me the API is what the plugin exposes at `server.plugins.task.api` similar to what store-server and account-server do. 

I would mention that hoodie-task-server also defines all necessary CouchDB replication routes so that task objects can be synced, maybe link to somewhere in the CouchDB docs where they are documented?